### PR TITLE
Document WEAVE_MTU in weave-kube config options

### PR DESCRIPTION
**Pull request by awh**

### CODE REVIEW

**Comment by awh** [comment]
#### Documentation changes:
modified [kube-addon](https://dev-weavewww.pantheonsite.io/docs/net/document-mtu-with-weave-kube/kube-addon/)

File: site/kube-addon.md [hunk 1]
```diff
@@ -144,3 +144,7 @@ The list of variables you can set is:
 * WEAVE\_EXPOSE\_IP - set the IP address used as a gateway from the
   Weave network to the host network - this is useful if you are
   configuring the addon as a static pod.
+* WEAVE\_MTU - Weave Net defaults to 1410 bytes, but you can set a
+  smaller size if your underlying network has a tighter limit, or set
+  a larger size for better performance if your network supports jumbo
+  frames.
```
Comment (rade) [inline site/kube-addon.md]: This should probably link to https://www.weave.works/docs/net/latest/using-weave/fastdp/ (ideally the sub-section that talks about MTUs, but it looks like we don't support sub-section linking, grrr).

Comment (bboreham) [inline site/kube-addon.md]: Sub-sections can be linked to if an anchor is created, e.g. like https://www.weave.works/docs/net/latest/using-weave/service-management/#routing

Comment (awh) [inline site/kube-addon.md]: > it looks like we don't support sub-section linking, grrr

PRs gratefully received at weaveworks/wordepress :smile:

**Comment by awh** [comment]
#### Documentation changes:
modified [kube-addon](https://dev-weavewww.pantheonsite.io/docs/net/document-mtu-with-weave-kube/kube-addon/)
modified [using-weave/fastdp](https://dev-weavewww.pantheonsite.io/docs/net/document-mtu-with-weave-kube/using-weave/fastdp/)

**Pull request merged**